src/Tizen.System.Resource/Resource/Resource.cs

```diff
@@ -0,0 +1,180 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.System
+{
+	/// <summary>
+	/// Provides methods to support CPU boosting and CPU inheritance.
+	/// </summary>
+    /// <since_tizen> 9 </since_tizen>
+	public class Resource
+	{
+        /// <summary>
+		/// Set cpu boosting for the target process (pid/tids).
+        /// </summary>
+		/// <since_tizen> 9 </since_tizen>
```

Comment:
If you have a plan to open this API as public API Level 10, 
```suggestion
		/// <since_tizen> 10 </since_tizen>
```